src/redux/rockets/rocket.js

```diff
@@ -0,0 +1,76 @@
+/* eslint-disable no-case-declarations */
```

Comment:
- [ ] it is always a good practice to make sure you fix all linter errors instead of disabling them, A linter is great for identifying errors when you use standard rules. Remember, a linter analyzes your code for stylistic and programming errors against the rules it knows. 💯 😉 